commit 48753b09ecce58700ca357a4ad0aa6bd6796c5b9 (refs/changes/28/5128/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-12-04T14:19:06-08:00 (10 months ago)
    
    MANTA-4033 get lagging Manta components on the latest v0.10.x
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/Makefile b/Makefile
index 14fa7d7..10f2638 100644
--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ JSSTYLE_FLAGS    = -f tools/jsstyle.conf
 # Variables
 #
 NAME                  = mola
-NODE_PREBUILT_VERSION = v0.10.32
+NODE_PREBUILT_VERSION = v0.10.48
 NODE_PREBUILT_TAG     = zone
 NODE_PREBUILT_IMAGE   = fd2cc906-8938-11e3-beab-4359c665ac99
 
